app/Forms/MembersSignupForm.php

```diff
@@ -100,7 +100,7 @@ protected function addSg() {
     protected function addMembership() {
         $this
             ->add('membership_type', 'choice', [
-                'choices' => array_combine(Member::$options_allowed_membership_type, Member::$options_membership_type_with_cost),
+                'choices' => Member::$options_membership_temp,
```

Comment:
Nit: bad variable naming, not descriptive -- temp. Avoid temp if possible, if not add a comment/TODO on why it's temp and what it needs to be in future.